website/api/zkvm/quickstart.md

```diff
@@ -5,96 +5,74 @@ slug: ./quickstart
 
 # zkVM Quick Start
 
-Welcome to the [zkVM] Quick Start page! Here are the steps between you and your first proof:
+Welcome to the zkVM Quick Start page! Here are the steps to create your first proof:
 
-1. Install the RISC Zero toolchain.
-2. Initialize a new project using the [cargo risczero] tool.
-3. Edit the boilerplate to build your first project.
-4. Run your project locally or with [Bonsai].
+1. Install the `risczero` toolchain
+2. Create a new project using the `risczero` tool.
+3. Familiarize yourself with the project's structure
+4. Run your project locally
+
+## Concept break
+
+zkVM provides a zero-knowledge proof of the correct execution of any Rust code.
+We want to run a piece of code and produce its proof of execution on zkVM. In the project code, we call zkVM a [prover]. The code we want to run on zkVM is a **[guest]** program. A proof of execution is a [receipt]; it contains a public part - a [journal] and an encryption part - a [seal].
+
+zkVM runs on the **[host]**, which loads the guest program to the zkVM and provides input to the guest. zkVM, or a prover, proves the guest execution by producing a locally verified receipt.
 
 ## 1. Install the RISC Zero Toolchain
 
-If you have already installed Rust via `rustup`, you can install our toolchain by running
+First, make sure you [install Rust], then install the RISC Zero toolchain by running the following:
 
 ```bash
 cargo install cargo-binstall
 cargo binstall cargo-risczero
 cargo risczero install
 ```
 
-If you need to install Rust or encounter problems, take a look at our [full installation instructions](install.md).
```

Comment:
It feels a bit odd to me to have a zkVM installation page and not refer to it in our zkVM quickstart. And that page does have more information than this one. If you want to drop this line can you reference those instructions in some other way?